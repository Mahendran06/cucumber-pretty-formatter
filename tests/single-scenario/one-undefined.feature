Feature: Single passing scenario
  In order to test event stream formatters
  As formatter
  I need to handle scenario with one undefined step

  Scenario: Scenario
    Then undefined
