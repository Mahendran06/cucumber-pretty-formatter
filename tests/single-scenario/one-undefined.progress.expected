U                                                                      1

1 scenarios (1 undefined)
1 steps (1 undefined)
0s (0B)

You can implement step definitions for undefined steps with these snippets:

func undefined() error {
	return godog.ErrPending
}

func FeatureContext(s *godog.Suite) {
	s.Step(`^undefined$`, undefined)
}

